server/views/pages/index.njk

```diff
@@ -3,72 +3,61 @@
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 
+{%- set dayjs = dayjs -%}
+{% set cases = unallocatedCases | length  %}
 {% set pageTitle = applicationName + " - Home" %}
+{% set tableData = {
+    caption: 'Unallocated community cases',
+    captionClasses: 'govuk-heading-l',
+    head: [
+        { text: 'Name / CRN' },
+        { text: 'Tier' },
+        { text: 'Sentence date' },
+        { text: 'Initial appointment' },
+        { text: 'Status' },
+        { text: 'Action' }
+    ],
+    rows: []
+} %}
+
+{% for item in unallocatedCases %}
+    {%- set tableRow = [
+        { text: item.name },
+        { text: item.tier },
+        { text: dayjs(item.sentenceDate).format('D MMM YYYY') },
```

Comment:
Would prefer this kind of prep to be done in the controller rather than passing whatever dayjs is into the template just to do this - I am a bit old-fashioned about views and like them to be as simple as possible.

Reply:
I'm going to add as a global nunjucks function as Carlo suggested as we'll be using throughout the app